### PR TITLE
526 update service history mg

### PR DESCRIPTION
**Pull request by markgreenburg**
- Updates json-schema dependency used on vets-website
- Updates reference for `servicePeriods` schema to actual `vets-json-schema` property

screen shot 2018-05-22 at 1 43 47 pm

screen shot 2018-05-22 at 1 43 30 pm

### CODE REVIEW

**Comment by peggygannon** [comment]
@markgreenburg drop downs look good.

File: yarn.lock [hunk 1]
```diff
@@ -9990,9 +9990,9 @@ verror@1.3.6:
   dependencies:
     extsprintf "1.0.2"
 
-"vets-json-schema@https://github.com/department-of-veterans-affairs/vets-json-schema.git#72b5c131b4a9ebd6b5773722ce8d923a86196ff2":
-  version "3.44.0"
-  resolved "https://github.com/department-of-veterans-affairs/vets-json-schema.git#72b5c131b4a9ebd6b5773722ce8d923a86196ff2"
+"vets-json-schema@https://github.com/department-of-veterans-affairs/vets-json-schema.git#86263a396bfe2b13153797bac78c424a913b450d":
+  version "3.46.0"
```
Comment (markgreenburg) [inline yarn.lock]: 3.45 was some validation updates that Rian pushed for another form. I've confirmed with him that we're good to skip up to 3.46, which contains these servicePeriod changes

**Review by cvalarida — APPROVED** [review]
I think this will have to be done again so we get `National Oceanic &amp; Atmospheric Administration` (with the `&amp;`) in there, but it looks good for now.

**Comment by cvalarida** [comment]
:thinking: Unless that's just a conversion the api will do? :man_shrugging: Either way, we need to make sure it gets sent to EVSS like that.

**Comment by markgreenburg** [comment]
@cvalarida that's a good point. `vets-api` will handle that conversion, both on the prefill side (@SMLuthi has already implemented this conversion for prefill) and on the submission side (documented in https://github.com/department-of-veterans-affairs/vets.gov-team/issues/10193 )

**Pull request merged**